aspnetcore/grpc/health-checks.md

```diff
@@ -60,7 +60,7 @@ The service name specified by the client is usually the default (`""`) or a pack
 
 ### Configure health checks execution interval
 
-Health checks are periodically executed using <xref:Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheckPublisher> to gather health results. By default, the publisher waits 5 seconds after app startup before running health checks, and then health checks are run again every 30 seconds.
+Health checks are executed immediately when `Check` is called. The `Watch` is a streaming method has different behavior. The long running stream reports health checks results over time by periodically executing <xref:Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheckPublisher> to gather health results. By default, the publisher waits 5 seconds after app startup before running health checks, and then health checks are run again every 30 seconds.
```

Comment:
```suggestion
Health checks are executed immediately when `Check` is called. `Watch` is a streaming method and has a different behavior than `Check`: The long running stream reports health checks results over time by periodically executing <xref:Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheckPublisher> to gather health results. By default, the publisher waits 5 seconds after app startup before running health checks, and then health checks are run again every 30 seconds.
```

Reply:
@JamesNK, this sentence had a typo I think.  Is this suggestion more along the lines of what you intended?